node/pkg/watchers/algorand/watcher.go

```diff
@@ -298,9 +299,15 @@ func (e *Watcher) Run(ctx context.Context) error {
 				}
 			}
 
+			if status.LastRound > math.MaxInt64 {
```

Comment:
Your error has uint64 vs. the check being on MaxInt64. Which one do you mean here?

Reply:
Good spot, updated the error string